testing/sakura/APKBUILD

```diff
@@ -0,0 +1,26 @@
+# Contributor: Niko Dittmann <mail@niko-dittmann.com>
+# Maintainer: Niko Dittmann <mail@niko-dittmann.com>
+pkgname=sakura
+pkgver="3.5.0"
+pkgrel=0
+pkgdesc="Sakura Terminal Emulator"
+url="https://launchpad.net/sakura"
+arch="all"
+license="GPL-2.0-only"
+makedepends="cmake vte3-dev pango-dev gtk+3.0-dev glib-dev libintl"
```

Comment:
Why do you have libintl here?
abuild needs headers to link to libraries which are provided by *-dev packages. *-dev packages depend on the actual libs (this case libintl) and pulls them in and links them.  After the build completes abuild will scan which libraries are linked in and detected which packages they belong to (provides) and depend on them.

libintl is part of gettext which has gettext-dev as headers. You should use this instead of libintl. Only in very rare cases libs should be explicitly depended.

Reply:
Thanks for the explanation. Changed & pushed.